Makefile

```diff
@@ -38,10 +38,17 @@ CPPFLAGS = -Wall -I$(SRC)include -std=$(STD) -pthread \
 ARFLAGS = rcs
 LDFLAGS = -pthread
 LDLIBS = $(TEST_LIBS)
+LN = ln
+CP = cp
+MKDIR = mkdir
 
 SUDO =
 
 CCTZ_LIB = libcctz.a
+CCTZ_STATIC_LIB = $(CCTZ_LIB)
+CCTZ_SHARED_LIB = libcctz.so.2.0
```

Comment:
Sure, usually a `VERSION` file is used. Would happily read that file to determine the version. I initially did a describe of the lastest `git tag` but realised that it wouldn't work for archives.